cmd/osbuild-composer/main.go

```diff
@@ -38,75 +39,84 @@ func main() {
 		}
 	}
 
-	log.Println("Loaded configuration:")
-	err = DumpConfig(config, log.Writer())
+	logrus.SetOutput(os.Stdout)
+	logLevel, err := logrus.ParseLevel(config.Logging.logLevel)
```

Comment:
Hm what will happen now if you don't explicitly specify the logging level? Maybe we could add the info level to `GetDefaultConfig`.

Reply:
I found out that default log level in logrus is `Info`, if we don't specify it in the config, then nothing happens here since it will be an empty string and not parse-able.

Reply:
Right, it's just relying on the parsing I guess to decide the default then. In that case the `Failed to set log level` is really expected behaviour though, and should just be an info message.